compute/rn/release-information/release-notes-22-06-update3.adoc

```diff
@@ -0,0 +1,59 @@
+:toc: macro
+== 22.06 Update 3 Release Notes
+
+The following table provides the release details:
+
+[cols="1,4"]
+|===
+|Build
+|22.06.XXX
+
+|Codename
+|Kepler, 22.06 Update 3
+//Tentative date
+|Release date
+|Nov 6, 2022
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|TBD
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+toc::[]
+
+=== Addressed Issues
+
+// GH#41077 PCSUP-11119
+* Improved the reconnection time for multi-tenant deployments when some tenants are disconnected from the central Console.
+
+// GH#40865	PCSUP-10977	
+* Fixed a DNS resolution error when running a twistcli image scan with the `--tarball` option.
+
+//GH#40694	PCSUP-10618	
+* Fixed an issue where errors were reported in scan results when the cloud service provider APIs are disabled.
++
+Now when the APIs for the service are disabled on the CSP, cloud discovery or registry scanning do not display these as errors in scan results. 
+The messages are added to the console logs.
+
+//GH#40533	PCSUP-10621	
+* Fixed an issue with incorrect cluster information in image scan results on "Monitor > Vulnerabilities > Deployed".
```

Comment:
"Monitor > Vulnerabilities > Deployed" as *Monitor > Vulnerabilities > Deployed*

Reply:
This syntax is accurate, please see https://redlock.atlassian.net/wiki/spaces/PCDTUS/pages/3728540016/Menucascade